### PR TITLE
[Copy] Update language evaluation labels

### LINKED ISSUE #7385: Language level text should use full names of test type
On the language information page of the profile. The different language levels are named a bit confusingly. For example, one of them is just "comprehension". We should switch to clearer names.

Current:
Image

Update to:
> Reading | Writing | Oral Interaction

French:
> Lecture | Écriture | Expression orale

## Acceptance Criteria

- [ ] labels update where language levels are used

### PR DESCRIPTION
**Pull request by vd1992**
🤖 Resolves #7385

## 👋 Introduction

Updates the labels as requested. 

## 🕵️ Details

Changed the label that also lists the three as I thought that was straightforward and should be in sync with the inputs. 
Also changed one place to reuse the labels from the utils file. 

## 🧪 Testing

1. Navigate to `/en/users/:id/profile#language-section`
2. Play with updating the language levels, observe the copy, and switch to French to check as well

## 📸 Screenshot

image

image

`/fr/admin/users/:id/profile#language-section`

image

### CODE REVIEW

**Comment by vd1992** [comment]
Storybook run

https://www.chromatic.com/review?appId=61e099a1bb1465003a73d3ce&number=7458&type=linked

**Comment by vd1992** [comment]
Oops, this label probably should've stuck around https://github.com/GCTC-NTGC/gc-digital-talent/pull/7458/commits/03385f9e26e8410b57e649962da2782943cc621e
Though, it is part of the old `LanguageInformationForm` which I don't think you can access anymore, conveniently.

File: apps/web/src/components/UserProfile/ProfileSections/LanguageInformationSection.tsx [hunk 1]
```diff
@@ -143,8 +143,8 @@ const LanguageInformationSection = ({
               <span data-h2-display="base(block)">
                 {intl.formatMessage({
                   defaultMessage:
-                    "Second language level (Comprehension, Written, Verbal):",
-                  id: "D7Qb41",
+                    "Second language level (Reading, Writing, Oral interaction):",
```
Comment (mnigh) [inline apps/web/src/components/UserProfile/ProfileSections/LanguageInformationSection.tsx:146]: Change to **Second language level (reading, writing, oral interaction)** and remove the colon from the string. Add the dividing colon message instead:

`{intl.formatMessage(commonMessages.dividingColon)}`

File: apps/web/src/lang/fr.json [hunk 2]
```diff
@@ -3884,6 +3880,10 @@
     "defaultMessage": "Compétences essentielles",
     "description": "Title text for group of need-to-have skills for pool advertisement on experience forms"
   },
+  "PZCSiW": {
+    "defaultMessage": "Niveau de connaissance de la langue seconde (Lecture, Écriture, Expression orale)",
```
Comment (mnigh) [inline apps/web/src/lang/fr.json:3884]:
```suggestion
    "defaultMessage": "Niveau de connaissance de la langue seconde (lecture, écriture, expression orale)",
```

File: apps/web/src/components/Profile/components/LanguageProfile/Display.tsx [hunk 3]
```diff
@@ -103,10 +103,10 @@ const Display = ({
         <FieldDisplay
           label={intl.formatMessage({
             defaultMessage:
-              "Second language level (Comprehension, Written, Verbal)",
-            id: "zF0F6w",
+              "Second language level (Reading, Writing, Oral interaction)",
```
Comment (mnigh) [inline apps/web/src/components/Profile/components/LanguageProfile/Display.tsx:106]: Change to **Second language level (reading, writing, oral interaction)** for  `defaultMessage` and `description`.

**Review by mnigh — CHANGES_REQUESTED** [review]
A few suggestions. Also, for consistency, could you also change Comprehension Level, Written Level, Verbal Level, and Second language level (Comprehension, Written, Verbal) in:

1. `usePoolCandidateCsvData`
2. `useUserCsvData`
3. `ProfileDocument`

## English then French
- Reading level = Niveau de lecture
- Writing level = Niveau d’écriture
- Oral interaction level = Niveau d’expression orale

**Comment by vd1992** [comment]
Found a bug too #7467

**Review by mnigh — APPROVED** [review]
Ça roule!

**Pull request merged**